packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -355,8 +355,15 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
 /// The possible states of a [PrimaryPointerGestureRecognizer].
 ///
 /// The recognizer advances from [ready] to [possible] when it starts tracking a
-/// primary pointer. When the primary pointer is resolved in the gesture
-/// arena (either accepted or rejected), the recognizers advances to [defunct].
+/// primary pointer. Where it advances from there depends on how the gesture is
+/// resolved for that pointer:
+///
+///  * If the primary pointer is resolved by the gesture winning the arena, the
+///    state remains as [possible] as long as the recognizer continues to track
```

Comment:
"remains as [possible] as long as" is confusing wording. Maybe "stays in the [possible] state for as long as" or something?